src/features/events_list/components/EventsPanel/EventsPanel.tsx

```diff
@@ -0,0 +1,87 @@
+import { Panel, PanelIcon } from '@konturio/ui-kit';
+import { useCallback, useState } from 'react';
+import clsx from 'clsx';
+import { Disasters24 } from '@konturio/default-icons';
+import { useAtom } from '@reatom/react';
+import { i18n } from '~core/localization';
+import { userResourceAtom } from '~core/auth';
+import { AppFeature } from '~core/auth/types';
+import { IS_MOBILE_QUERY, useMediaQuery } from '~utils/hooks/useMediaQuery';
+import { useAutoCollapsePanel } from '~utils/hooks/useAutoCollapsePanel';
+import { panelClasses } from '~components/Panel';
+import { useHeightResizer } from '~utils/hooks/useResizer';
+import { MIN_HEIGHT } from '../../constants';
+import { ShortState } from '../ShortState/ShortState';
+import { FullState } from '../FullState/FullState';
+import s from './EventsPanel.module.css';
+import type { ShortStateListenersType } from '@konturio/ui-kit/tslib/Panel';
```

Comment:
it should be `from '@konturio/ui-kit'`;
it's not exported in index.ts in ui repository

Reply:
will be fixed with component update